app/models/term_fingerprint.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: term_fingerprints
+#
+#  id          :integer          not null, primary key
+#  fingerprint :string
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#
+class TermFingerprint < ApplicationRecord
```

Comment:
Now that we have decided to only have one set of stored fingerprints, it feels worth considering whether to name this `Fingerprint` instead of `TermFingerprint`. This is non-blocking, but if you choose to not make this change I'd like to hear why you prefer it this way.

As noted elsewhere, I'd prefer if we can move the logic to calculate a fingerprint to this model rather than keep it in Term.

Reply:
Do you have thoughts about naming the class versus naming the field? Using the same name for both feels like it would get awkward, particularly from the perspective of the Term model. `term.fingerprint` feels like the method to call to get the calculated string, but if the class is also Fingerprint then things get a bit weird.

For now I'm using `.fingerprint_value` as the Term method, but I'm not sure that's going to be the case when it gets to code review.

Reply:
One thing I'm particularly interested in your feedback about is the naming being used here. Having a `Fingerprint` model, and a `fingerprint` field, feels awkward, particularly from the Term perspective. I've tried to resolve this by adding the `.fingerprint_value` construct that delegates down to the Fingerprint model, and allows the Term to just get access to the string value somehow.

That said, I don't think this gets used anywhere yet - so maybe it would makes sense to remove it until the UI or something leverages it.